src/pages/Staking/index.tsx

```diff
@@ -559,6 +559,8 @@ const Staking = (): JSX.Element => {
     );
   };
 
+  const hasStakedAmount = stakedAmount?.gt(ZERO_GOVERNANCE_TOKEN_AMOUNT);
```

Comment:
Since `stakedAmount?.gt(ZERO_GOVERNANCE_TOKEN_AMOUNT)` is repeated, a variable `hasStakedAmount` is introduced.